mayastor/src/bdev/storage/iscsi.rs

```diff
@@ -0,0 +1,167 @@
+use std::{
+    collections::HashMap,
+    convert::TryFrom,
+    ffi::CString,
+    os::raw::{c_int, c_void},
+};
+
+use async_trait::async_trait;
+use futures::channel::oneshot;
+use snafu::ResultExt;
+use url::Url;
+use uuid::Uuid;
+
+use spdk_sys::{create_iscsi_disk, delete_iscsi_disk, spdk_bdev};
+
+use crate::{
+    bdev::util::uri,
+    core::Bdev,
+    ffihelper::{cb_arg, done_errno_cb, errno_result_from_i32, ErrnoResult},
+    nexus_uri::{self, BdevCreateDestroy},
+};
+
+use super::StorageBdev;
+
+const ISCSI_IQN_PREFIX: &str = "iqn.1980-05.mayastor";
+
+#[derive(Debug)]
+pub struct Iscsi {
```

Comment:
Back to the naming thing, it was always iSCSI if I believe but the namespace was different, right?

it used to be Bdev::Iscsi or Bdev::Aio -- now its bdev::storage::Iscsi